coordinator/src/test/scala/com/socrata/datacoordinator/service/collocation/CoordinatedCollocatorTest.scala

```diff
@@ -314,11 +347,45 @@ class CoordinatedCollocatorTest extends FunSuite with Matchers with MockFactory
       result.id should be (None)
       result.status should be (Approved)
       result.message should be (Approved.message)
-      result.cost should be (Cost(2))
+      result.cost should be (Cost(1))
 
-      result.moves.size should be (2)
-      result.moves.map(_.datasetInternalName).toSet.size should be (1)
-      result.moves.map(_.storeIdTo).size should be (2)
+      result.moves.size should be (1)
+      val move = result.moves.head
+      Set(move.storeIdFrom, move.storeIdTo) should be (Set(store1A, store5A))
+    }
+  }
+
+  test("explainCollocation for non-collocated datasets on the same stores should generate no moves") {
+    withMocks(Set(storeGroupA), { coordinator =>
+      (coordinator.secondaryGroups _).expects().returns(Map(storeGroupA -> groupConfig(2, storesGroupA)))
+
+      (coordinator.secondaryMoveJobs _).expects(storeGroupA, alpha1).once.returns(moveJobResultEmpty)
+      (coordinator.secondariesOfDataset _).expects(alpha1).once.returns(secondaries(store1A, store3A))
+
+      (coordinator.secondaryMoveJobs _).expects(storeGroupA, bravo1).once.returns(moveJobResultEmpty)
+      (coordinator.secondariesOfDataset _).expects(bravo1).once.returns(secondaries(store1A, store3A))
+
+      expectCDOIDatasetsWithNoCollocations(coordinator, Set(alpha1)) // group for alpha.1
+      expectCDOIDatasetsWithNoCollocations(coordinator, Set(bravo1)) // group for bravo.1
+    }) { case (collocator, _) =>
+      collocator.explainCollocation(storeGroupA, request(Seq((alpha1, bravo1)))) should be (resultEmpty)
+    }
+  }
+
+  test("explainCollocation for already collocated datasets should return a result with complted status") {
```

Comment:
nit: `complted`

Reply:
(I'll fix in another pull request)

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/debd0644-c4b9-4157-9c4d-b4dea6359bc4-69d0c8e1-e986-47fb-a727-1a8ae0328e06-v1.png?transparent=1&palette=1&width=246)